mlir/lib/Dialect/Arith/Utils/Utils.cpp

```diff
@@ -294,6 +302,13 @@ Value mlir::createScalarOrSplatConstant(OpBuilder &builder, Location loc,
   return builder.createOrFold<arith::ConstantOp>(loc, type, splat);
 }
 
+Type mlir::getIntType(OpFoldResult ofr) {
```

Comment:
Nit: I think it might be better to rename this just `getType` ?

Reply:
Actually looking at its uses, you dont need the `getIntType` at all. You can just fold this into `getValueOrCreateConstantIntOp`?